<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThingsWidget</class>
 <widget class="QWidget" name="ThingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="ThingsView" name="thingsTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thingsSummaryLayout">
     <item>
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>Всего предметов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelThingsCountAll">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_76">
       <property name="text">
        <string>на сумму:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelThingsPriceAll">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_18">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="moneySummaryLayout">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Всего дринков:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMoneySummary">
       <property name="text">
        <string>n/a</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_45">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ThingsView</class>
   <extends>QTableView</extends>
   <header>thingstab/thingsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
